syntax = "proto3";

message SensorBaro {
  int64 timestamp_us = 1; // Timestamp in microseconds
  float pressure_pa = 2; // Pressure in Pascals
  float temperature_c = 3; // Temperature in Celsius
  float altitude_m = 4; // Estimated altitude in meters
}